wmf/uninstall.md

```diff
@@ -1,8 +1,8 @@
-# Uninstallation Instructions
+# Uninstalling WMF v5
```

Comment:
This change is not necessary as the uninstallation instructions are in context of WMF 5 only.